pkg/statik/fs/fs.go

```diff
@@ -243,14 +220,23 @@ func registerCustomExternal(cache Cache, opt AssetOption) error {
 			opt.Shasum, sum, assetURL)
 	}
 
-	if storeInCache {
-		cache.Set(key, unzippedData, cacheTTL)
-	} else {
-		cache.RefreshTTL(key, cacheTTL)
+	asset := newAsset(opt, zippedDataBuf.Bytes(), unzippedData)
+
+	objectName := path.Join(asset.Context, asset.Name)
+	swiftConn := config.GetSwiftConnection()
+
+	f, err := swiftConn.ObjectCreate(DynamicAssetsContainerName, objectName, true, "", "", nil)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
```

Comment:
We need to check if `f.Close()` has been successful or if an error has been returned.